src/schema/v2/fields/money.ts

```diff
@@ -131,10 +131,34 @@ export const Money = new GraphQLObjectType<any, ResolverContext>({
       type: new GraphQLNonNull(GraphQLFloat),
       description:
         "An amount of money expressed in major units (like dollars).",
-      resolve: ({ cents, currency }) => {
+      args: {
+        convertTo: {
+          type: GraphQLString,
+          description: "ISO-4217 code of a destination currency for conversion",
+        },
+      },
+      resolve: async (
+        { cents, currency },
+        { convertTo },
+        { exchangeRatesLoader }
+      ) => {
         const factor = currencyCodes[currency.toLowerCase()].subunit_to_unit
         // TODO: Should we round or used a fixed precision?
-        return cents / factor
+        const major = cents / factor
+
+        const needsConversion = !!convertTo && convertTo !== currency
+        if (needsConversion) {
+          const exchangeRates = await exchangeRatesLoader()
+          // from here, very USD specific
+          if (convertTo !== "USD") {
```

Comment:
This can maybe move above the `if (needsConversion)`, just to save a fetch if an unsupported conversion is requested.